.github/workflows/test.yml

```diff
@@ -45,11 +45,6 @@ jobs:
       - name: Install Composer dependencies
         run: composer install
 
-      # Scan the logs for failing tests and surface that information by creating annotations and log file decorations.
-      - name: Setup problem matcher to provide annotations for PHPUnit
-        # The JSON file is provided by the `shivammathur/setup-php` action. See https://github.com/shivammathur/setup-php#problem-matchers.
-        run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
-
```

Comment:
How is this fixing the attribute value per 9969e8ae479465b42e648870ae0d4607223799ca?

Reply:
The PHPUnit matcher annoyed me and made looking through the PR unnecessarily difficult. Besides, it almost always just points into the `SpecTest.php` file, so it's useless in multiple ways, not just one. :)

Reply:
But I should have added it in a separate commit, I agree. I probably just didn't notice that these changes were part of that commit.